src/Components/NavBar/NotificationsBadge.tsx

```diff
@@ -38,61 +40,75 @@ export const NotificationsBadge: React.FC = () => {
           0
         )
 
-        let notifications = totalUnread
+        let count = totalUnread
 
         // Update the notification bad with the count, and store it in a cookie
         // so that subsequent page views don't need a fetch in order to render
         // the badge.
-        if (notifications > 0) {
+        if (count > 0) {
           const cachedNotificationCount = Number(
             cookie.get("notification-count")
           )
-          if (notifications !== cachedNotificationCount) {
-            if (notifications >= 100) {
-              notifications = "99+"
+          if (count !== cachedNotificationCount) {
+            if (count >= 100) {
+              count = "99+"
             }
 
             // In force, when a request is made to `/notifications` endpoint,
             // sd.NOTIFICATIONS_COUNT is populated by this cookie.
-            cookie.set("notification-count", notifications)
+            cookie.set("notification-count", count)
           }
         }
 
         // User has no notifications; clear the cookie
-        if (notifications === 0) {
+        if (count === 0) {
           cookie.expire("notification-count")
           return null
         }
 
-        const trackOnHover = once(() => {
-          trackEvent({
-            subject: AnalyticsSchema.Subject.NotificationBell,
-            new_notification_count: totalUnread,
-          })
-        })
-
         return (
-          <Box onMouseOver={trackOnHover}>
-            <CircularCount notifications={notifications} />
+          <Box>
+            <CircularCount count={count} rawCount={totalUnread} hover={hover} />
           </Box>
         )
       }}
     />
   )
 }
 
-const CircularCount: React.FC<{ notifications?: string }> = ({
-  notifications,
-}) => {
+const CircularCount: React.FC<{
+  /**
+   * Formatted count for display
+   */
+  count?: string
+  /**
+   * Raw unread count, used for analytics.
+   */
+  rawCount?: boolean
```

Comment:
Is this `number`?

Reply:
Weird! how did this slip through

Reply:
Ahh, I know how -- the fallback, 0, is falsy:

<img width="214" alt="Screen Shot 2019-05-21 at 11 58 18 AM" src="https://user-images.githubusercontent.com/236943/58123068-d429f400-7bbf-11e9-8938-149919367920.png">

Fixed by adding a type